src/ObjectListView3/ObjectListView.py

```diff
@@ -548,6 +555,42 @@ def AddObjects(self, modelObjects):
         finally:
             self.Thaw()
 
+    def AddObjects(self, modelObjects):
+        warnings.warn(
+            "'AddObjects()' is deprecated, use 'ExtendObjects()'",
+            DeprecationWarning)
+        self.ExtendObjects(modelObjects)
+
+    def InsertObject(self, modelPosition, modelObject, before=False):
+        """
+        Insert the given object after/before the position of the insert object.
+        """
+        self.InsertObjects(modelPosition, [modelObject], before=before)
+
+    def InsertObjects(self, modelPosition, modelObjects, before=False):
+        """
+        Insert the given objects after/before the position of the insert object.
+        """
+        if len(self.innerList) == 0 and len(self.modelObjects) == 0:
+            return self.SetObjects(modelObjects)
+
+        try:
+            self.Freeze()
+            originalSize = len(self.innerList)
```

Comment:
@sphh Is `originalSize` supposed to be used somewhere in this code?

Reply:
That line can be removed!